submissions/BlueLamantine/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,58 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/BlueLamantine/a-tiny-JS-world
+   Web app: https://bluelamantine.github.io//a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+const man = {
+   species: 'Human',
+   name: 'Parker',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'What a wonderful world!',
+   friends: ['Eva','Hachiko']
+};
+ const dog = {
+   species: 'Dog',
+   name: 'Hachiko',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Woof-woof',
+   friends: ['Parker']
+};
+const woman = {
+   species: 'Human',
+   name: 'Eva',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Feminism in the air!',
+   friends: ['Parker', 'Garfield']
+};
+const cat = {
+   species: 'Cat',
+   name: 'Garfield',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Meoooooow',
+   friends: ['Eva','Parker']
+};
+const catWoman = {
+   species: 'Supernatural',
+   name: 'Mary',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: cat.saying,
+   friends: 'No friends'
+};
+const jsWorld = [man,woman,dog,cat,catWoman];
+// ======== OUTPUT ========
```

Comment:
I think this comment is redundant here. To my mind, it's clear and it doesn't need to be commented

Reply:
ok